projects/robots/husarion/rosbot_xl/protos/RosbotXl.proto

```diff
@@ -0,0 +1,253 @@
+#VRML_SIM R2023b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# This is a proto file for Webots for the RosbotXl
+EXTERNPROTO "webots://projects/devices/slamtec/protos/RpLidarA2.proto"
+EXTERNPROTO "webots://projects/devices/tdk/protos/Mpu-9250.proto"
+
+PROTO RosbotXl [
+  field  SFVec3f     translation     0 0 0
+  field  SFRotation  rotation        0 0 1 0
+  field  SFString    name            "rosbot_xl" # Is `Robot.name`.
+  field  SFString    controller      "rosbot_xl" # Is `Robot.controller`.
+  field  MFString    controllerArgs  []          # Is `Robot.controllerArgs`.
+  field  SFString    customData      ""          # Is `Robot.customData`.
+  field  SFBool      supervisor      FALSE       # Is `Robot.supervisor`.
+  field  SFBool      synchronization TRUE        # Is `Robot.synchronization`.
+  field  SFBool      selfCollision   FALSE       # Is `Robot.selfCollision`.
+  field  MFNode      lidarSlot       [ RpLidarA2 { name "laser" } ] # Extends the robot with a RpLidar A2 lidar.
```

Comment:
same applies to this lidar

Reply:
Same case here we, use in [ros_components_description](https://github.com/husarion/ros_components_description/blob/ros2/urdf/slamtec_rplidar_a2.urdf.xacro) we use `laser` as a frame name of scan.